rest/client.py

```diff
@@ -7,28 +8,39 @@ class APIV2Client:
     def __init__(self, client_id: str = None, api_key: str = None, api_secret: str = None):
         pass
 
+    def _make_request(self, method: str, endpoint: str, data: typing.Dict = None):
```

Comment:
Okay, this actually isn't a bad idea at all. 

Also, lets generalize method even more by having (instead of data):

params: typing.Dict = None (this would represent query parameters)
body: typing.Dict = None (this would represent body for more complex requests like POST / PUT)

One another thing: You ALWAYS want to put private methods under public API of the package.
Hence, please move _make_request method under all public functions.

Reply:
I have renamed data to params, added body.
_make_requests moved to the bottom of the package. 

Please resolve if you see fit.